app/views/admin/merchants/index.html.erb

```diff
@@ -22,7 +22,9 @@
         </td>
         <td>
           <% if merchant.enabled? %>
-            <%= link_to 'Disable Merchant', admin_merchant_path(merchant.id), method: :patch %>
+            <%= button_to 'Disable Merchant', admin_merchant_path(merchant.id), method: :patch %>
+          <% else %>
+            <%= button_to 'Enable Merchant', admin_merchant_path(merchant.id), method: :patch %>
```

Comment:
Nice! I was going to suggest that this be a button instead and you read my mind! 🧠